.github/workflows/ci.yml

```diff
@@ -20,6 +20,8 @@ jobs:
           - 4.4
           - 4.5
           - 4.6
+          - 4.7
+          - 4.8
```

Comment:
Also need some new exclusions:

```yaml
          - { ruby-version: 2.3, redis-version: 4.5 }     # already excluded
          - { ruby-version: 2.3, redis-version: 4.6 }     # already excluded
          - { ruby-version: 2.3, redis-version: 4.7 }
          - { ruby-version: 2.3, redis-version: 4.8 }
          - { ruby-version: 2.3, redis-version: latest }  # already excluded
          - { ruby-version: 2.4, redis-version: latest }
```

Reply:
Thanks. Will get that hammered out soon.